### PR TITLE
csds client: add type resolver support for envoy.extensions.load_balancing_policies.*

### PR DESCRIPTION
**Pull request by Rainton**
Add type resolver support for [envoy.extensions.load_balancing_policies.*](https://github.com/envoyproxy/envoy/tree/main/api/envoy/extensions/load_balancing_policies) protos to parse them correctly.

Signed-off-by: Yutong Li [yutongli@google.com](mailto:yutongli@google.com)

### CODE REVIEW

File: csds-client/client/util/util.go [hunk 1]
```diff
@@ -323,7 +340,8 @@ func GenerateGraph(data GraphData) (string, error) {
 
 // OpenBrowser opens url in browser based on platform
 // TODO: the url cannot be passed correctly on some platforms because of \" and ",
-//  which need to be solve in the future.
+//
+//	which need to be solve in the future.
```
Comment (adisuissa) [inline csds-client/client/util/util.go:344]: nit: remove empty line.

**Review by adisuissa — COMMENTED** [review]
LGTM, Thanks!
One small nit.

**Review by adisuissa — APPROVED** [review]
LGTM, thanks!
@htuch for a final pass.

**Review by htuch — APPROVED** [review]
LGTM, thanks!

**Pull request merged**